qiskit_dynamics/models/lindblad_model.py

```diff
@@ -0,0 +1,654 @@
+    matrix commutator and anti-commutator, respectively. In the above:
+
+        - :math:`H(t)` denotes the Hamiltonian,
+        - :math:`L_j` denotes the :math:`j^{th}` dissipator, or Lindblad,
+          operator, and
+        - :math:`\gamma_j(t)` denotes the signal corresponding to the
+          :math:`j^{th}` Lindblad operator.
+
+    Instantiating an instance of :class:`~qiskit_dynamics.models.LindbladModel`
+    requires specifying the above decomposition:
+
+    .. code-block:: python
+
+        lindblad_model = LindbladModel(static_hamiltonian=static_hamiltonian,
+                                       hamiltonian_operators=hamiltonian_operators,
+                                       hamiltonian_signals=hamiltonian_signals,
+                                       static_dissipators=static_dissipators,
+                                       dissipator_operators=dissipator_operators,
+                                       dissipator_signals=dissipator_signals)
+
+    where the arguments ``hamiltonian_operators``, ``hamiltonian_signals``, and
+    ``static_hamiltonian`` are for the Hamiltonian decomposition as in
+    :class:`~qiskit_dynamics.models.HamiltonianModel`,
+    and the ``dissipator_operators`` correspond to the :math:`L_j`, and the ``dissipator_signals``
+    the :math:`g_j(t)`, which default to the constant ``1.``.
+    """
+
+    def __init__(
+        self,
+        static_hamiltonian: Optional[Union[Array, csr_matrix]] = None,
+        hamiltonian_operators: Optional[Union[Array, List[csr_matrix]]] = None,
+        hamiltonian_signals: Optional[Union[List[Signal], SignalList]] = None,
+        static_dissipators: Optional[Union[Array, csr_matrix]] = None,
+        dissipator_operators: Optional[Union[Array, List[csr_matrix]]] = None,
+        dissipator_signals: Optional[Union[List[Signal], SignalList]] = None,
+        rotating_frame: Optional[Union[Operator, Array, RotatingFrame]] = None,
+        in_frame_basis: bool = False,
+        evaluation_mode: Optional[str] = "dense",
+        validate: bool = True,
+    ):
+        """Initialize.
+
+        Args:
+            static_hamiltonian: Constant term in Hamiltonian.
+            hamiltonian_operators: List of operators in Hamiltonian with time-dependent
+                                   coefficients.
+            hamiltonian_signals: Time-dependent coefficients for hamiltonian_operators.
+            static_dissipators: List of dissipators with coefficient 1.
+            dissipator_operators: List of dissipator operators with time-dependent coefficients.
+            dissipator_signals: Time-dependent coefficients for dissipator_operators.
+            rotating_frame: Rotating frame in which calcualtions are to be done.
+                            If provided, it is assumed that all operators were
+                            already in the frame basis.
+            in_frame_basis: Whether to represent the model in the basis in which the rotating
```

Comment:
What happens if you supply a rotating frame, but don't put "in_frame_basis"? Do we need `in_frame_basis` as a kwarg or should it automatically be set to `True` if you supply a rotating frame and then you can set it to false after construction if you don't want to go into that frame?